jekyll/_cci2/sample-config.md

```diff
@@ -31,6 +31,12 @@ This document provides sample [`.circleci/config.yml`]({{ site.baseurl }}/config
 * TOC
 {:toc}
 
+CirlceCI has created an **extension for Visual Studio Code** that reduces context switching for developers between the web app and VS Code through a set of helpful features.
+
+The VS Code extension reduces the time to create, modify, and troubleshoot configuration files through real-time syntax validation, highlighting, and autocomplete suggestions. Authenticating the extension with your CircleCI account will provide you with status alerts as well as the ability to validate your configuration by testing the pipeline locally before you push your code. 
```

Comment:
Hi! 
> as well as the ability to validate your configuration by testing the pipeline locally before you push your code.

I am afraid this isn't exact, I'm sorry if this was extracted by information I provided. 
Can I suggest rephrasing to this?
 
> Authenticating the extension with your CircleCI account will also allow you to visualise and manage your CircleCI pipelines directly from your IDE, and be notified of workflow status changes.

Reply:
@BeFunes just some clarification on my end - what is this extension actually compatible with? Because Visual Studio has VS Code - a code editor, as well as Visual Studio IDE - and IDE. I just want to be careful with how we're using "editor" vs "IDE" when discussing this since Visual Studio has both sets of tools.